src/App.js

```diff
@@ -5,20 +5,7 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <h1>Inicio do Projeto!</h1>
```

Comment:
Acho que ficaria melhor desta forma que tal ?
```suggestion
        <h1>Front end online store!</h1>
```